modules/storage-expanding-add-volume-expansion.adoc

```diff
@@ -12,7 +12,14 @@ have the `allowVolumeExpansion` field set to `true`.
 
 .Procedure
 
-* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute.
+* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute by running the following command:
```

Comment:
Just checking - is the update to this file something you were expecting, or is it an accidental inclusion from someone else's? It doesn't seem related to your other update, and the commit is shared between you and courtney.

Reply:
Accidental inclusion. Thanks for finding it.